lib/updateVersion.js

```diff
@@ -980,3 +987,151 @@ function glTF10to20(gltf) {
   // Remove empty arrays
   removeEmptyArrays(gltf);
 }
+
+// It's not possible to upgrade glTF 1.0 shaders to 2.0 PBR materials in a generic way,
+// but we can look for certain uniform names that are commonly found in glTF 1.0 assets
+// and create PBR materials out of those.
+const baseColorTextureNames = ["u_tex", "u_diffuse"];
+const baseColorFactorNames = ["u_diffuse"];
+
+function initializePbrMaterial(material) {
+  material.pbrMetallicRoughness = defined(material.pbrMetallicRoughness)
+    ? material.pbrMetallicRoughness
+    : {};
```

Comment:
Only reason is to avoid the allocation if the left hand side is already defined. Though this code is only called once so `defaultValue` would not cause much of a GC hit. I might just leave it as-is though both approaches are fine here.